packages/api/package.json

```diff
@@ -24,6 +24,16 @@
         "types": "./lib/internal.d.ts",
         "default": "./lib/internal.js"
       }
+    },
+    "./decorator": {
```

Comment:
Should this be... `/internal/decorator`?

Reply:
If we were re-export it from other place (like component, maybe). For `botframework-webchat-api` it isn't really internal compared to other things the package exposes